### DISCUSSION TITLE
pnp-modern-search-parts-v4.sppkg

### DISCUSSION THREAD

**Discussion by cnsieler** [microsoft-search/pnp-modern-search#4079] [Q&A] [answered]
I am dumber than a post.  I downloaded the Zip file.  [PnP Modern Search - 4.13.1](https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.13.1).  I am trying to find the pnp-modern-search-parts-v4.sppkg (Mandatory). to upload.  i can not find it.  I know it is something simple so please have patience.

**Comment by kasperbolarsen** [comment]
Like how to install it? Please see https://microsoft-search.github.io/pnp-modern-search/installation/

**Comment by cnsieler** [comment]
I got all that I can not find the Package to Up load in what I downloaded.
All I fing is file folders and json files

**Comment by cnsieler** [comment]
And all the instructions say[image: image.png]
 is download the package and Upload into sharepoint.

I have unzipped and locked for the package in the files -
[image: image.png]

**Comment by cnsieler** [comment]
Where is this located Add *pnp-modern-search-parts-v4.sppkg* to the global
tenant app catalog or a site collection app catalog. If you don't have an
app catalog, follow this procedure
<https://docs.microsoft.com/en-us/sharepoint/use-app-catalog> to create one.

**Reply by kasperbolarsen** [reply, marked as answer]
So the question is where the current version of the package is? 
https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.13.1

**Comment by cnsieler** [comment]
Here are the steps I took following the instructions above.
I went to the link above to download.
I think I solved it.  The answer is scroll down the page at the Link above.
Then I noticed this https://github.com/microsoft-search/pnp-modern-search/releases/download/4.13.1/pnp-modern-search-parts-v4.sppkg
You did help you forced me to see what I was doing wrong so thank you kind friend.

**Reply by kasperbolarsen** [reply]
You can find a lot of good getting-started videos on YouTube and a couple of Modern Search courses as well

**Comment by cnsieler** [comment]
Thanks, I'll check them out.
